src/backend/commands/matview.c

```diff
@@ -3303,11 +3303,6 @@ clean_up_IVM_hash_entry(MV_TriggerHashEntry *entry, bool is_abort)
 		}
 		if (!is_abort)
 		{
-			if (table->slot)
-			{
-				ExecDropSingleTupleTableSlot(table->slot);
-				table->slot = NULL;
-			}
```

Comment:
Are there any problems?  the codes looks good:)

Reply:
Hold on , I just test this on CI , but CI is broken.

Reply:
<img width="716" alt="image" src="https://github.com/user-attachments/assets/e92b1316-7615-4a43-ae34-ad6a2a8fe0b7">

```c++
if (resowner)
		{
			ResourceOwnerRelease(resowner,
								RESOURCE_RELEASE_BEFORE_LOCKS,
								false, /* isCommit */
								false); /* isTopLevel */
			ResourceOwnerRelease(resowner,
								RESOURCE_RELEASE_LOCKS,
								false, /* isCommit */
								false); /* isTopLevel */
			ResourceOwnerRelease(resowner,
								RESOURCE_RELEASE_AFTER_LOCKS,
								false, /* isCommit */
								false); /* isTopLevel */
		}
```
release resource in ResourceOwnerRelease is better.  So del ExecDropSingleTupleTableSlot.